node.sh

```diff
@@ -0,0 +1,53 @@
+package: node
+version: "v9.8.0"
+build_requires:
+  - curl
+prefer_system: "(?!slc5)"
+prefer_system_check: |
+  which node || { echo "node is missing"; exit 1; }
```

Comment:
I know it sounds weird but we should not assume `which` exists. Please try to invoke `node` directly, in a way that it returns 0: for instance, `node -h &> /dev/null` instead of `which node` would do (I have checked).

Reply:
Okay, but there are ~20 recipes using `which` including O2 so I assume it must exist.

Reply:
Sure but don't please :-) The other 20 recipes are "wrong"(-ish). It's in any case better to invoke the command directly if you can.

Reply:
ok done